.ci_support/migrations/pytorch26.yaml

```diff
@@ -0,0 +1,10 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for pytorch 2.6
+  kind: version
+  migration_number: 1
+libtorch:
+- '2.6'
+pytorch:
+- '2.6'
+migrator_ts: 1739565601.4436605
```

Comment:
Given we already include PyTorch 2.6 in [the global pinnings]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/43c9c1b866a01c18e5700521a05ca75915e59447/recipe/conda_build_config.yaml#L882-L883 ), guessing we don't need the PyTorch 2.6 migrator any more